src/query/interpreter.cpp

```diff
@@ -2892,7 +2903,7 @@ Interpreter::PrepareResult Interpreter::Prepare(const std::string &query_string,
 
     return {query_execution->prepared_query->header, query_execution->prepared_query->privileges, qid};
   } catch (const utils::BasicException &) {
-    EventCounter::IncrementCounter(EventCounter::FailedQuery);
+    memgraph::metrics::IncrementCounter(memgraph::metrics::FailedQuery);
```

Comment:
What is exactly semantic of failed query? Wrong query typed or

Reply:
I think yes, the query has not started executing but while parsing or semantics. E.g. syntaxexception or semanticexception.

Reply:
That's fine but why is then the check in interpreter.hpp?

Reply:
@as51340, what do you mean here by semantic? `FailedQuery` is used for both parsing and execution error

Reply:
Yes then it makes sense, I just wanted to check which errors are included